### PR TITLE
feat: export compile function by default, also return normalized spec

### PR DESCRIPTION
**Pull request by domoritz**
Now this works

```js
var {spec, normalized} = vegaLite(vlSpec, options);
```

in addition to

```js
var {spec} = vegaLite.compile(vlSpec, options);
```

### CODE REVIEW

File: src/vl.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/**
```
Comment (kanitw) [inline src/vl.ts]: Having this file is weird. You can just do the following in index.ts.

```
/**
function compileToSpec(inputSpec: TopLevelSpec, opt: CompileOptions = {}) {
  return compile(inputSpec, opt).spec;
}


export default Object.assign(compileToSpec, {compile, extractTransforms, normalize, version});
```

**Comment by kanitw** [comment]
Yeah, maybe let's not do it. :)

**Comment by domoritz** [comment]
I think that `compile.compile` is not all that useful since it always only returns an object with the spec (#5287). I'd like to deprecate that function. In this PR, I attempted to keep a backward-compatible version around.

**Comment by domoritz** [comment]
Updated. I have not added a new compile function but just export the compile function by default and also make it return the normalized spec.

**Comment by domoritz** [comment]
Replaced by https://github.com/vega/vega-lite/pull/5555

**Pull request closed without merging**